src/main/java/com/ericsson/ei/waitlist/WaitListWorker.java

```diff
@@ -73,7 +70,7 @@ public void run() {
                     checkTargetAggregationsExistAndRepublishEvent(eventJson);
                 }
             } catch (Exception e) {
-                LOGGER.error("Exception occured while trying to resend event: {}", document, e);
+                LOGGER.error("Exception occured while trying to resend event: " + document, e);
```

Comment:
Please change back to use {} and ,

Reply:
Yes, use parameterized logging instead of concat.

Reply:
This could be due to merging.

Reply:
changed